src/frontend/planner_test/tests/testdata/expr.yaml

```diff
@@ -351,3 +351,22 @@
   batch_plan: |
     BatchProject { exprs: ['integer[][]':Varchar] }
     └─BatchValues { rows: [[]] }
+- name: now expression
```

Comment:
We only test the happy path, but I'm afraid of pushing down conjunctions that include now, especially pushing it onto a join's condition or a scan operator's filter... Then the checks on LogicalFilter -> StreamFilter will not run, and we will end up with an expression in the operator that cannot be evaluated...?

Reply:
I guess maybe a function call to `now` will not get pushed down ever?

Reply:
> checks on LogicalFilter -> StreamFilter will not run, and we will end up with an expression in the operator that

fixed